pkg/cli/config/config.go

```diff
@@ -90,11 +90,10 @@ func GetBackplaneConfiguration() (bpConfig BackplaneConfiguration, err error) {
 		}
 	}
 
-	// proxyURL is required
-	proxyInConfigFile := viper.GetStringSlice("proxy-url")
-	proxyURL := bpConfig.getFirstWorkingProxyURL(proxyInConfigFile)
-	if proxyURL != "" {
-		bpConfig.ProxyURL = &proxyURL
+	// Load proxy URLs as a slice of strings
+	bpConfig.ProxyURLs = viper.GetStringSlice("proxy-url")
+	if len(bpConfig.ProxyURLs) > 0 {
+		bpConfig.ProxyURL = &bpConfig.ProxyURLs[0]
```

Comment:
does this mean we only use the first specified proxy?

Reply:
> does this mean we only use the first specified proxy?

I am going to revert this part as below to only keep the `proxyURL` field and updated my `CheckProxyConnectivity()` function accordingly.
```
// proxyURL is required
	proxyInConfigFile := viper.GetStringSlice("proxy-url")
	proxyURL := bpConfig.getFirstWorkingProxyURL(proxyInConfigFile)
	if proxyURL != "" {
		bpConfig.ProxyURL = &proxyURL
	}
```
So, we will only use the **first working proxy** defined in the local backplane config.